tsconfig.lib.json

```diff
@@ -22,7 +22,8 @@
     "allowSyntheticDefaultImports": true,
     "resolveJsonModule": true,
     "paths": {
-      "@core/*": ["packages/core/src/*"]
+      "@core/*": ["packages/core/src/*"],
+      "@dev-tools": ["packages/dev-tools/src/index.ts"]
```

Comment:
Typescript won't be able to generate types from alias paths, use relative paths like we do everywhere.

Reply:
We should think about whether we should add the devtools as a dependency or let the user explicitly add it and initialize it themselves.

What do you think?